tokio-console/src/config.rs

```diff
@@ -240,6 +291,7 @@ impl FromStr for LogFilter {
 struct ConfigFile {
     default_target_addr: Option<String>,
     log: Option<String>,
+    warns: Vec<KnownWarnings>,
```

Comment:
let's rename this field to
```suggestion
    warnings: Vec<KnownWarnings>,
```
as "warns" is not the correct pluralization.

Reply:
Updated. https://github.com/tokio-rs/console/pull/493/commits/23cdd53fe1d1f5b7fbc37ca86aa9f15992c9c41b